docs/getting-started-with-react/getting-started-with-react.stories.mdx

```diff
@@ -183,3 +183,43 @@ The final part of our example defines the custom actions needed to power our
 component. In this case the action when the user clicks on the **Search**
 button. Here we call the `setURL` setter to show the `loadingGIF`. Then, we call
 `setURL` awaiting the async response of `getGIF`.
+
+## Types of Components in Atlantis and their Ability for Customization
+
+#### 1. Primitive Components:
+
+Examples: [Icon](..?path=/docs/components-images-and-icons-icon--docs),
+[Avatar](..?path=/docs/components-images-and-icons-avatar--docs)
+
+These components do not accept children and have minimal customization.
+
+#### 2. Simple Components:
+
+Examples: [Button](..?path=/docs/components-actions-button--docs),
+[Link](..?path=/docs/components-text-and-typography-link--docs)
+
+These allow basic customization through props or limited children, like
+`ReactNode` or specific child types.
+
+#### 3. Compound Components:
+
+Examples: [Chip](..?path=/docs/components-selections-chip--docs),
+[DataList](..?path=/docs/components-lists-and-tables-datalist--docs)
```

Comment:
DataList is almost in its own category. it's bordering on an entire feature/view.

I suppose it is technically a compound component though. it's just the largest compound component we have.